listenbrainz/domain/critiquebrainz.py

```diff
@@ -1,21 +1,29 @@
 import time
-from datetime import timezone
 
+import json
+import requests
 from requests_oauthlib import OAuth2Session
 from oauthlib.oauth2.rfc6749.errors import InvalidGrantError
+from typing import Iterable
 
 from data.model.external_service import ExternalServiceType
+from listenbrainz.db.model.review import CBReviewMetadata
 from listenbrainz.db import external_service_oauth
 
 from flask import current_app
 
 from listenbrainz.domain.external_service import ExternalService, ExternalServiceInvalidGrantError
 
+
 CRITIQUEBRAINZ_SCOPES = ["review"]
 
 OAUTH_AUTHORIZE_URL = "https://critiquebrainz.org/oauth/authorize"
 OAUTH_TOKEN_URL = "https://critiquebrainz.org/ws/1/oauth/token"
 
+CRITIQUEBRAINZ_REVIEW_SUBMIT_URL = "https://critiquebrainz.org/ws/1/review/"
+CRITIQUEBRAINZ_REVIEW_FETCH_URL = "https://beta.critiquebrainz.org/ws/1/reviews/"
```

Comment:
This PR depends on https://github.com/metabrainz/critiquebrainz/pull/383 . A reminder to me to change this to cb prod once that PR is merged and deployed.